ethdb/pebble/pebble.go

```diff
@@ -338,6 +338,16 @@ func (d *Database) Delete(key []byte) error {
 	return d.db.Delete(key, nil)
 }
 
+// DeleteRange removes all keys in the range [start,end) from the key-value store.
```

Comment:
Perhaps make it a bit clearer by reusing the doc from pebble:
```suggestion
// DeleteRange deletes all of the keys (and values) in the range [start,end)
// (inclusive on start, exclusive on end).
```

Reply:
done